pyproject.toml

```diff
@@ -68,3 +71,11 @@ extend-exclude = '''
   | reflex-generator
 )
 '''
+
+[tool.isort]
+line_length = 120
+profile = "black"
```

Comment:
What does this profile setting do? Does it link defaults to black? That would be ideal.

Maybe we should write it as 'black' for consistency with the other settings?

Reply:
https://pycqa.github.io/isort/docs/configuration/black_compatibility.html

For future reference